plugin-server/src/worker/ingestion/event-pipeline/populateTeamDataStep.ts

```diff
@@ -53,6 +54,13 @@ export async function populateTeamDataStep(
             teamResolutionChecksCounter.labels({ check_ok: checkOk }).inc()
             // statsd copy as prometheus is currently not supported in worker threads.
             runner.hub.statsd?.increment('ingestion_team_resolution_checks', { check_ok: checkOk })
+            if (!checkOk) {
```

Comment:
I am very confused by js but: 

```js
Welcome to Node.js v18.14.0.
Type ".help" for more information.
> const checkOk = 'false'
undefined
> !checkOk
false
```

I think we should do a different check here.

Reply:
oh, good catch! thanks!

Reply:
I guess it's consistent:

```js
> const checkOk = 'true'
undefined
> !checkOk
false
```

Reply:
Yup, that's a "truthie" check, non-empty string is `truthy`